package.json

```diff
@@ -45,20 +55,30 @@
   },
   "homepage": "https://github.com/SignHash/signhash-contracts#readme",
   "dependencies": {
-    "truffle": "^4.0.1",
-    "ethereumjs-testrpc": "^6.0.1",
-    "npm-run-all": "^4.1.1"
+    "ethereumjs-testrpc": "^4.1.1",
+    "mkdirp": "^0.5.1",
+    "truffle-artifactor": "^3.0.0",
+    "truffle-compile": "^3.0.2",
+    "truffle-config": "^1.0.3",
+    "truffle-migrate": "^2.0.0",
+    "truffle-resolver": "^4.0.1"
   },
   "devDependencies": {
     "@types/bignumber.js": "^4.0.3",
     "@types/chai": "^4.0.4",
-    "@types/mocha": "^2.2.43",
+    "@types/es6-promise": "0.0.33",
+    "@types/mkdirp": "^0.5.1",
+    "@types/mocha": "^2.2.44",
+    "@types/node": "^8.0.51",
     "@types/ramda": "^0.24.14",
+    "chai": "^4.1.2",
     "husky": "^0.14.3",
-    "lint-staged": "^4.2.2",
+    "lint-staged": "^4.3.0",
+    "npm-run-all": "^4.1.2",
     "prettier": "^1.7.0",
     "ramda": "^0.24.1",
     "solhint": "^1.1.7",
+    "truffle": "^4.0.1",
     "tslint": "^5.7.0",
     "typescript": "^2.5.2"
   }
```

Comment:
So, unfortunately, removing the `migrate` and moving `truffle` to `devDependencies` breaks the webapp build pipeline. Webapp relies on being able to run `contracts:migrate` as a single command, when truffle is in dev dependencies it's not installed when installing `signhash-contracts` and thus it is impossible to run migrations via CLI.